packages/react/src/primitives/types/stepperField.ts

```diff
@@ -57,9 +57,7 @@ export interface StepperFieldProps extends TextFieldProps {
 
   /**
    * @description
-   * TODO:
-   * Extends StepperField props from Omit<TextFieldProps, 'onChange'>, after removing [key: string]: any from the base type
-   * and rename onStepChange to onChange
```

Comment:
This was being rendered in the docs props tables

Reply:
🤦‍♂️